lib/Data/Database.php

```diff
@@ -768,17 +835,37 @@ private static function _createCommentTable()
     private static function _createConfigTable()
     {
         list($main_key, $after_key) = self::_getPrimaryKeyClauses('id');
+        $charType                   = self::$_type === 'oci' ? 'VARCHAR2(16)' : 'CHAR(16)';
+        $textType                   = self::_getMetaType();
         self::$_db->exec(
-            'CREATE TABLE ' . self::_sanitizeIdentifier('config') .
-            " ( id CHAR(16) NOT NULL$main_key, value TEXT$after_key );"
+            'CREATE TABLE "' . self::_sanitizeIdentifier('config') .
+            "\" ( \"id\" $charType NOT NULL$main_key, \"value\" $textType$after_key )"
         );
         self::_exec(
-            'INSERT INTO ' . self::_sanitizeIdentifier('config') .
-            ' VALUES(?,?)',
+            'INSERT INTO "' . self::_sanitizeIdentifier('config') .
+            '" VALUES(?,?)',
             array('VERSION', Controller::VERSION)
         );
     }
 
+    /**
+     * sanitizes CLOB values used with OCI
+     *
+     * From: https://stackoverflow.com/questions/36200534/pdo-oci-into-a-clob-field
```

Comment:
Hmm as in the original PR I guess PHPDoc [needs the link to be defined as a link to work](https://stackoverflow.com/questions/7692309/phpdoc-inline-link-and-netbeans):
```suggestion
     * from {@link https://stackoverflow.com/questions/36200534/pdo-oci-into-a-clob-field}
```

Reply:
I used the "From:" as it is the convention in our current codebase. It seems to link just fine, see for example https://privatebin.info/codedoc/classes/PrivateBin.I18n.html#method_loadTranslations - we can change it, but I suggest we open another merge request for that and do them all at once?

Reply:
Ah, then fine, yeah. Likely the theme or phpdoc generator just detects it is a link and anyway links it. Good then…

Yeah in PHPDoc there is also `@seealso` I guess, that would be the bette rthing and yeah for a new PR then.